ui/raidboss/data/04-sb/raid/o6s.txt

```diff
@@ -3,9 +3,10 @@
 hideall "--Reset--"
 hideall "--sync--"
 
-0.0 "--Reset--" sync / 21:........:4000000F:/ window 100000 jump 0
+# Note: Although there is a 18 sec. delay between the chatlog line and the boss being targetable
```

Comment:
Ah, nice.  Did you go test this, I'm guessing?

Reply:
> Ah, nice. Did you go test this, I'm guessing?

Yup.  There were a number of other omega fights that were also started based on chatlog lines - o7, o8, o12s, maybe another one or two.  I swapped in the 0x104 sync and saw in-game that it was causing the first sync ability to jump forward by what seemed to be about ~0.5s.  (e.g. a 12.5 sync would actually seem to sync at 12.0, causing the timeline to jump forward by 0.5).  But I reverted and noticed some variability even using the chatlog lines.

I also compared some of the 0x104 timelines using make_timeline against what we already have, and the drift was actually much smaller (~0.1s/0.2s).  Given that, I *think* we're probably okay to keep the timelines as-is and not regenerate them based on the new 0x104 start?  🙏

Reply:
Yeah, I think we're ok.  I usually (try to) double check timelines when I write them to make sure they're when combat is starting and not based on the auto or an inconsistent chat log time.  If anything is broken, we can just fix that when it comes up.